### ISSUE TITLE
Stored seed / password didn't carry over on last update

### ISSUE THREAD

**Issue by richper413** [iotaledger/android-wallet-app#118]
Description:
---

Updated the app and my stored seed / password was gone when I opened it. I think I have the correct seed saved in a file but I'm showing a 0 balance and I can't see any of my previous transactions. I found my coins on the tangle explorer but I don't know how to recover it.

Device info:
---
<table>
<tr><td>App version</td><td>0.9.1</td></tr>
<tr><td>App version code</td><td>9100</td></tr>
<tr><td>Android build version</td><td>G935VVRU4BQK2</td></tr>
<tr><td>Android release version</td><td>7.0</td></tr>
<tr><td>Android SDK version</td><td>24</td></tr>
<tr><td>Android build ID</td><td>NRD90M.G935VVRU4BQK2</td></tr>
<tr><td>Device brand</td><td>Verizon</td></tr>
<tr><td>Device manufacturer</td><td>samsung</td></tr>
<tr><td>Device name</td><td>hero2qltevzw</td></tr>
<tr><td>Device model</td><td>SM-G935V</td></tr>
<tr><td>Device product name</td><td>hero2qltevzw</td></tr>
<tr><td>Device hardware name</td><td>qcom</td></tr>
<tr><td>ABIs</td><td>[arm64-v8a, armeabi-v7a, armeabi]</td></tr>
<tr><td>ABIs (32bit)</td><td>[armeabi-v7a, armeabi]</td></tr>
<tr><td>ABIs (64bit)</td><td>[arm64-v8a]</td></tr>
</table>

**Comment by pinpong** [comment]
Hi, after a tangle snapshot all previous trx gone and saved within a new "genesis" file. Please create new addresses until you balance is back or check the link below.


https://blog.iota.org/gui-wallet-phase-two-of-the-reclaim-process-f5913109cf46

**Comment by richper413** [comment]
I don't think what I saved is the seed. I think it is just the first transaction I made to the android wallet from my Bitfinex account. When I use it as my seed, I've generated several more addresses than I ever did in transactions (only had a the 1 incoming transaction and 2 outgoing) I still see no balance. I've rebroadcasted all of them several times but only 1 has ever shown confirmed but it has a 0 balance attached. Am I basically screwed?
